packages/flutter_tools/test/general.shard/macos/cocoapods_test.dart

```diff
@@ -52,7 +51,6 @@ void main() {
 
   String podsIsInCustomDir({String cocoapodsReposDir}) {
     cocoapodsReposDir ??= fileSystem.path.join(
-      globals.fsUtils.homeDirPath,
```

Comment:
This test apparently doesn't rely on this value, avoid looking up potentially null home dir path